cishouseholds/derive.py

```diff
@@ -39,18 +39,18 @@ def assign_datetime_from_coalesced_columns_and_log_source(
         if type != "timestamp":
```

Comment:
Please drop the parsing part of the function - it should expect all date/datetime fields to be timestamp type already